scipy-ar-2013/videos/scipyconar-2013-inteligencia-artificial-con-python-pedro-fisanotti-dia-1-de-3.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "",
```

Comment:
I wonder if we cannot use the data from http://scipyla.org/conf/2013/activities/index.html
```
Día Uno:
(T) ¿Por qué Inteligencia Artificial?
Ejemplos de problemas para despertar el interés en el tema.
Elección de un problema particular para aplicar todo lo que se vaya viendo a lo largo del taller.
Introducción al concepto de Inteligencia Artificial, enfoques posibles, rama elegida para el taller.
(T) Agentes racionales, tipos de ambientes, limitaciones.
(T) Noción básica de lo que es un problema de búsqueda. Diferencia entre buscar un estado y buscar un camino de acciones.
(T+P) Formulación de problemas de búsqueda (elementos que debemos entender y definir de nuestro problema).
Representación del estado.
Definición de las acciones posibles desde cada estado.
Función de costo.
Función de comprobación de meta o función de valoración.
Función heurística.
(T) Algoritmos para resolver problemas de búsqueda. Idea general y diferentes tipos.
(T) Presentación de la biblioteca a usar en la práctica: SimpleAI. Ventajas de usar python para este tipo de programación.
(P) Instalación de las herramientas necesarias en las pcs de los asistentes.
```